### ISSUE TITLE
can't generate Release-iphoneos

### ISSUE THREAD

**Issue by myie9** [Carthage/Carthage#3091]
* carthage install method: homebrew
* `which carthage`:
* `carthage version`: 0.17.2
* `xcodebuild -version`: Xcode 11.5
Build version 11E608c
* Are you using `--no-build`? no
* Are you using `--no-use-binaries`? no
* Are you using `--use-submodules`? no
* Are you using `--cache-builds`? no
* Are you using `--new-resolver`? no

**Cartfile**
```
<YOUR CARTFILE>
```
github "getsentry/sentry-cocoa"             ~> 4.2
github "SnapKit/SnapKit"                    ~> 5.0
github "SDWebImage/SDWebImage"              ~> 5.0
github "swisspol/GCDWebServer"              ~> 3.3
github "kif-framework/KIF"                  "v3.7.11"
github "mozilla-mobile/onepassword-app-extension"    "add-framework-support"
github "jrendel/SwiftKeychainWrapper"       ~> 3.2.0
github "DaveWoodCom/XCGLogger"              ~> 7.0
github "cezheng/Fuzi"                       ~> 3.0
github "SwiftyJSON/SwiftyJSON"              ~> 5.0
github "mozilla-mobile/telemetry-ios"      ~> 1.1
github "mozilla-mobile/MappaMundi"          "master"
github "Leanplum/Leanplum-iOS-SDK"          ~> 2.7.2
github "mozilla-services/shavar-prod-lists" "72.0"

# Use beta version of A~S for XCode12
github "nbhasin2/application-services" "63.0b"
# Use release version of application-services
# commenting out until we have a fix for https://github.com/mozilla/application-services/issues/3670
# github "mozilla/application-services"      "v65.0.0"

# Use interim binary version of application-services
# binary "https://circleci.com/api/v1.1/project/github/mozilla/application-services/2862/artifacts/0/dist/mozilla.app-services.json" ~> 0.0.1-snapshot # mozilla/application-services@fb5c540e99133980911216055425688f22a27be2

**Carthage Output**
```
<OUTPUT>
```
FengdeMacBook-Pro-3:firefox-ios fengguo$ ./carthage_command.sh 
Xcode SDK: 13
*** xcodebuild output can be found in /var/folders/g0/fg976j3s577_9th4qqllw8lc0000gn/T/carthage-xcodebuild.1ztX5k.log
*** Fetching MappaMundi
*** Building scheme "AStar iOS" in AStar.xcodeproj
Failed to write to /Users/fengguo/tmp/firefox-ios/Carthage/Build/iOS/AStar.framework: Error Domain=NSCocoaErrorDomain Code=260 "The file “AStar.framework” couldn’t be opened because there is no such file." UserInfo={NSFilePath=/Users/fengguo/Library/Developer/Xcode/DerivedData/AStar-enkvjetujlscybgjpvyxzvxdgyzu/Build/Products/Release-iphoneos/AStar.framework, NSUnderlyingError=0x7fe916645910 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
FengdeMacBook-Pro-3:firefox-ios fengguo$ ls /Users/fengguo/Library/Developer/Xcode/DerivedData/AStar-enkvjetujlscybgjpvyxzvxdgyzu/Build/Products
Release-iphonesimulator

**Actual outcome**
Carthage did not generate Release-iphoneos. but only generate Release-iphonesimulator

**Expected outcome**
Carthage should generate Release-iphoneos. 
How can I make it generate generate Release-iphoneos?

**Comment by tmspzz** [comment]
Please fix your formatting. It's really hard to follow the issue this way.

Have you checked the pinned issue about Xcode 12 work around?
